kubernetes/coredns.yaml.sed

```diff
@@ -58,7 +58,7 @@ data:
         kubernetes CLUSTER_DOMAIN REVERSE_CIDRS {
           pods insecure
           upstream
-          fallthrough in-addr.arpa ip6.arpa
+          FALLTHROUGH
```

Comment:
I guess this change will fail the Configmap translation test

Reply:
yes - since we are comparing full configmap literals in that test